src/Lesson1/Lesson1.java

```diff
@@ -0,0 +1,94 @@
+package Lesson1;
+
+public class Lesson1 {
+
+    public static void main(String[] args) {
+        type();
+
+        result(105, 34, 25, 5);
+
+        boolean result2 = predel2(20, 15);
+        System.out.println(result2);
+
+        type_number(5);
+
+        System.out.println(type_number2(-5));
+
+        name("Коля");
+
+        year(2019);
+
+    }
+
+    static void type(){
+        byte b1 = 100;
+        short s1 = -12059;
+        int i1 = 1235425978;
+        long l1 = 23589317895L;
+        float f1 = 125.28f;
+        double d1 = 75328.237;
+        char c1 = 'a';
+        boolean b2 = true;
+        System.out.println(b1);
+        System.out.println(s1);
+        System.out.println(i1);
+        System.out.println(l1);
+        System.out.println(f1);
+        System.out.println(d1);
+        System.out.println(c1);
+        System.out.println(b2);
+    }
+
+    static void result(float a, float b, float c, float d){
+        System.out.println(a * (b + (c / d)));
+    }
+
+    static boolean predel2(float n, float m){
+        boolean t = true;
+        boolean fal = false;
+        if ((n+m)>=10 && (n+m)<=20){
+            return t;
+        } else {
+            return fal;
+        }
+    }
```

Comment:
```suggestion
        return ((n+m)>=10 && (n+m)<=20);
    }
```
аналогично и в других местах